### PR TITLE
Arrowtable

### PR DESCRIPTION
**Pull request by rcdenyer**
Need to remember to add 'USE arrowdb' to top of sql file when regenerating with sequel pro!

### CODE REVIEW

File: data/data.sql [hunk 1]
```diff
@@ -34,7 +34,7 @@ CREATE TABLE `archer` (
   `dob` date NOT NULL,
   `country` varchar(255) DEFAULT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=latin1;
```
Comment (John-A-J) [inline data/data.sql]: please return to previous

File: data/data.sql [hunk 2]
```diff
@@ -64,17 +154,17 @@ CREATE TABLE `tournament` (
   `type` varchar(255) DEFAULT NULL,
   `arrows` int(11) unsigned DEFAULT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=latin1;
```
Comment (John-A-J) [inline data/data.sql]: please undo this change

File: data/data.sql (hunk 1 above)
Comment (Rohoolio) [inline data/data.sql]: Shall we just check this in case?
